ground_truth_labeling_jobs/multi_modal_parallel_sagemaker_labeling_workflows_with_step_functions/sagemaker_ground_truth_workflows.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
<p>Let's add more context so people know why this notebook is useful:</p><p> </p><blockquote>Data labeling often requires a single data object to include multiple types of annotations, or multi-type, such as 2D boxes (bounding boxes), lines, and segmentation masks, all on a single image. Additionally, to create high-quality machine learning (ML) models using labeled data, you need a way to monitor the quality of the labels. You can do this by creating a workflow in which labeled data is audited and adjusted as needed. This notebook introduces a solution to address both of these labeling challenges using an automotive dataset, and you can extend this solution for use with any type of dataset.</blockquote><p> </p><p>Additionally, we must tell customers that this notebook only works if they have already launched the cloud formation stack in provided in the blog post. If possible, let's also provide a link to launch the stack here as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>